File: Course_01/C Language/EstruturaRepetitiva/senha_fixa.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int senha, senhaCorreta;

    printf("Digite a senha: ");
    scanf("%d", &senha);

    senhaCorreta = 2002;

    while (senha != senhaCorreta) {
        printf("Senha Invalida! Tente novamente: ");
        scanf("%d", &senha);
    }
    return 0;
}

File: Course_01/Java Language/Estrutura repetitiva/senha_fixa.java
package Repetitivo;

import java.util.Locale;
import java.util.Scanner;

public class senha_fixa {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int senha, tentativa;
		
		System.out.print("Digite a senha: ");
		tentativa = sc.nextInt();
		
		senha = 2002;
		
		while (tentativa != senha) {
			System.out.print("Senha Invalida! Tente novamente: ");
			tentativa = sc.nextInt();
		}
		
		System.out.println("Acesso permitido!");
		
		sc.close();
	}

}
